runtime/descriptor.go

```diff
@@ -117,5 +119,31 @@ func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object
 		}
 		return ret, raised
 	}
+	if writtable {
+		setterFunc := func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+			ret := None
+			var raised *BaseException
+			if raised = checkFunctionArgs(f, fieldName, args, ObjectType, ObjectType); raised == nil {
+				self := args[0]
+				newvalue := args[1]
+				if !newvalue.isInstance(nativeFieldType) {
```

Comment:
I guess so, but may be less readable. Anyway I had followed the ``getterFunc()`` style. I could change mine or both. Please confirm.